android/src/main/java/ly/count/dart/countly_flutter/CountlyFlutterPlugin.java

```diff
@@ -135,6 +135,13 @@ public final int resultResponder(RequestResult rResult) {
 
     List<CountlyFeedbackWidget> retrievedWidgetList = null;
 
+    //-------------TESTING RELATED------------------
+    private static Boolean isAppStartTimeTracked = false;
+    private static Boolean isFBEnabled = false;
```

Comment:
isForegroundBackgroundEnabled is shorther than the other 
isManualAppLoadedTriggerEnabled

I think this reduces readibility